Sources/BraintreePayPal/BTPayPalRequest.swift

```diff
@@ -179,6 +184,10 @@ import BraintreeCore
         if let userPhoneNumberDict = try? userPhoneNumber?.toDictionary() {
             parameters["phone_number"] = userPhoneNumberDict
         }
+        
+        if let shopperSessionID {
+            parameters["shopper_session_id"] = shopperSessionID
```

Comment:
@scannillo @sarahkoop do we also need to be setting the `apiClient.metadata.sessionID` as this value as well in the `BTPayPalClient`? Was just thinking if a merchant uses 2 different `BTAPIClient`s for those flows the PayPal flow I believe would randomly generate a new session ID vs reusing the session ID from the `BTShopperInsightsClient`?

Reply:
Ah yes I think we will want to update to set the sessionID for PayPal too

Reply:
I think it would be easier to tackle that in separate tickets though (will be needed for Android too)

Reply:
Do we want to update in this same PR or create another ticket to update PayPal?

Reply:
I would vote a separate ticket and separate PRs

Reply:
What do y'all think about just using a new tag for `shopper_session_id`? The existing `session_id` is very SDK specific and IMO will be confusing to start mixing with shopper session ID for some flows, but not all